docs/usage/AdjustingControllerLimits.md

```diff
@@ -0,0 +1,28 @@
+# Adjusting controller limits
+The controller has memory and CPU limits specified to prevent it from using an excess of resources. However, it may be necessary to adjust these upwards or downwards, especially if i restarts because it is out of memory (OOM).
```

Comment:
```
However, it may be necessary to adjust these upwards or downwards, especially if i restarts because it is out of memory (OOM).
```
>
```
It's possible to adjust memory limits, which may be useful if you notice the controller restarting due to OOMKilled errors.
```